general/communication.hpp

```diff
@@ -19,31 +19,30 @@
 #include "array.hpp"
 #include "table.hpp"
 #include "sets.hpp"
+#include "globalostream.hpp"
+#include "globalcomm.hpp"
 #include <mpi.h>
 
+
 namespace mfem
 {
 
 /** @brief A simple convenience class that calls MPI_Init() at construction and
     MPI_Finalize() at destruction. It also provides easy access to
-    MPI_COMM_WORLD's rank and size. */
+    to the global_mpi_comm rank and size. */
```

Comment:
`MFEM_COMM_WORLD` instead of `global_mpi_comm`?

Reply:
Done in Veselin's update.